### PR TITLE
Destroy method

### PR DESCRIPTION
**Pull request by rgolea**
This should allow you to destroy the current view and remove all event listeners to avoid memory leaks.

Usage:
```
var Graph = require('p2p-graph')

var graph = new Graph('.root')

graph.destroy();
```

### CODE REVIEW

**Review by roccomuso — CHANGES_REQUESTED** [review]
The code is not standard-compliant.
Make sure to follow the JS standard rules, otherwise CI tests are not going to pass.

**Comment by rgolea** [comment]
@roccomuso where should I see that? Why is it not standard-compliant?

**Comment by roccomuso** [comment]
@rgolea 
```
  /home/travis/build/feross/p2p-graph/index.js:425:50: Extra semicolon.
  /home/travis/build/feross/p2p-graph/index.js:427:56: Extra semicolon.
  /home/travis/build/feross/p2p-graph/index.js:429:19: Missing space before function parentheses.
  /home/travis/build/feross/p2p-graph/index.js:429:21: Missing space before opening brace.
  /home/travis/build/feross/p2p-graph/index.js:430:33: Extra semicolon.
  /home/travis/build/feross/p2p-graph/index.js:431:18: Extra semicolon.
  /home/travis/build/feross/p2p-graph/index.js:432:41: Extra semicolon.
  /home/travis/build/feross/p2p-graph/index.js:433:61: Extra semicolon.
```
Just get rid of those semicolons and add a space before the function parentheses and the opening brace.
This is our code style: [standard JS](https://github.com/feross/standard).

**Comment by rgolea** [comment]
@roccomuso changed! Thank you!

**Comment by roccomuso** [comment]
@rgolea one last change required:
`p2p-graph/index.js:429:22: Missing space before opening brace.`

**Comment by rgolea** [comment]
Sorry! Haven't used standardjs before.

**Review by roccomuso — APPROVED** [review]

**Comment by rgolea** [comment]
Should this have a version bump?

**Comment by roccomuso** [comment]
@rgolea Don't worry.
Semver version bump:
```
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.
```
This should be a minor update. But I'd wait before a version bump.
Let's wait for a review from @feross

**Comment by rgolea** [comment]
@roccomuso, @feross Thank you!

File: package.json
Comment (feross) [inline package.json:4]: Please leave the version bump to the maintainers. Otherwise, what happens if we merge another PR first and that one becomes 1.1.2? Then the version bump in this PR will be incorrect. Best to not touch the version.

**Review by feross — APPROVED** [review]
LGTM, but in the future, please do not change the version in `package.json` or commit the built files (`p2p-graph.min.js`) as that part is done by the maintainer right before publishing a new version.

**Comment by rgolea** [comment]
@feross I haven't really worked that much in the community as open source. Thank you very much for the feedback!

**Comment by feross** [comment]
@rgolea No problem, thanks for the PR 👍

**Pull request merged**
